json-rpc/types/src/views.rs

```diff
@@ -641,6 +644,8 @@ pub enum VMStatusView {
     VerificationError,
     DeserializationError,
     PublishingFailure,
+    #[serde(other)]
+    Unknown,
```

Comment:
Should we have this on all serialized enums?

Reply:
These are only really necessary for client facing things (also I don't think this works with bcs?)